src/components/FeaturePanel/FeaturePanel.tsx

```diff
@@ -29,7 +28,7 @@ const Flex = styled.div`
   flex: 1;
 `;
 
-export const FeaturePanel = () => {
+export const FeaturePanel = forwardRef<HTMLDivElement>((_, ref) => {
```

Comment:
Can you please use `headingRef` prop instead?

`forwardRef` is meant to be used for "the main ref of a component", eg. for `FeaturePanel` it would be some div for the whole panel. I may be mistaken, feel free to look it up. 🙂